addons/website_sale/tests/test_sale_process.py

```diff
@@ -74,6 +75,11 @@ def test_03_demo_checkout(self):
         self.start_tour("/", 'shop_buy_product', login="demo")
 
     def test_04_admin_website_sale_tour(self):
+        self.env['website'].get_current_website().company_id = self.env.company
```

Comment:
```suggestion
        self.env['website'].browse(1).company_id = self.env.company
```

`get_current_website()` if not in a frontend context (MockRequest or tour or controller) is not supposed to be called, it will most probably return something but it is not supposed to and is not very reliable if not in the mentioned context.

Most of the tests in website related modules just hardcode `1` (both in `browse()` or `with_context(website_id=1)`)